i18n/en-US/common.json

```diff
@@ -0,0 +1,55 @@
+{
+    "REQUEST_ERROR": "An error occurred making that request.",
+    "INVALID_OPTION": "Invalid option given.",
+    "NONE": "None",
+    "USER_FETCH_ERROR": "An error occured while trying to fetch the requested user.",
+    "ALERT_SYSTEM": "TypicalBot Alert System",
+    "MODERATOR_FIELD": "» Moderator",
+    "REASON_FIELD": "» Reason",
+    "NO_REASON": "No reason provided.",
+    "USER_NOT_FOUND": "The requested user could not be found.",
+    "WRONG_VOICE": "You must be in the same voice channel to perform that command.",
+    "NEED_QUEUE": "This command only works while in queue mode.",
+    "NO_RESPONSE": "No response was given.",
+    "SUCCESS": "Success.",
+    "COMMANDS": "Commands",
+    "INVITES": "Invites",
+    "STARS": "Stars",
+    "NA": "n/a",
+    "CHANNEL": "Channel",
+    "ENABLE": "Enable",
+    "ENABLED": "Enabled",
+    "DISABLE": "Disable",
+    "DISABLED": "Disabled",
+    "CLEAR": "Clear",
+    "ADD": "Add",
+    "REMOVE": "Remove",
+    "HERE": "Here",
+    "DEFAULT": "Default",
+    "EMBED": "Embed",
+    "NOTHING_STREAMING": "Nothing is currently streaming.",
+    "CANCELED": "Canceled.",
+    "RESET": "Reset",
+    "CHANGED": "Changed",
+    "NAME_FIELD": "» Name",
+    "ID_FIELD": "» ID",
+    "OWNER_FIELD": "» Owner",
+    "CREATED_FIELD": "» Created",
+    "REGION_FIELD": "» Region",
+    "VERIFICATION_FIELD": "» » Verification Level",
+    "CHANNELS_FIELD": "» Channels",
+    "MEMBERS_FIELD": "» Members",
+    "ROLES_FIELD": "» Roles",
+    "EMOJIS_FIELD": "» Emojis",
+    "BOOSTED_FIELD": "» Boosted Tier",
+    "DISPLAYNAME_FIELD": "» Display Name",
+    "STATUS_FIELD": "» Status",
+    "JOINED_FIELD": "» Joined",
+    "REGISTERED_FIELD": "» Registered",
+    "TOTALVIEWS": "» Total Views",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.